zeppelin-zengine/src/main/java/org/apache/zeppelin/interpreter/InterpreterFactory.java

```diff
@@ -672,7 +672,7 @@ public void createInterpretersForNote(InterpreterSetting interpreterSetting, Str
     Interpreter interpreter;
     for (InterpreterInfo info : interpreterInfos) {
       if (option.isRemote()) {
-        if (option.isConnectExistingProcess()) {
+        if (option.isExistingProcess()) {
```

Comment:
I think `InterpreterOption.isConnectExistingProcess()` is no longer used in anywhere in the code base.
To avoid confusion between `InterpreterOption.isConnectExistingProcess()` and `InterpreterOption. isExistingProcess()`, @astroshim could you remove `InterpreterOption.isConnectExistingProcess()` method?